packages/zwave-js/src/lib/controller/ReplaceFailedNodeRequest.ts

```diff
@@ -68,6 +68,7 @@ export class ReplaceFailedNodeRequestBase extends Message {
 interface ReplaceFailedNodeRequestOptions extends MessageBaseOptions {
 	// This must not be called nodeId or rejectAllTransactions may reject the request
 	failedNodeId: number;
+	includeNonSecure: boolean;
```

Comment:
nope, that doesn't belong here

Reply:
I'm still not sure about this :confused:

Reply:
See other comment above. The message classes do not care about this property, since it is not exchanged with the stick. The controller itself needs to remember that the node it is going to include should be included securely.